pom.xml

```diff
@@ -691,12 +698,13 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.eclipse.jetty</groupId>
-                <artifactId>jetty-maven-plugin</artifactId>
+                <groupId>org.eclipse.jetty.ee10</groupId>
+                <artifactId>jetty-ee10-maven-plugin</artifactId>
                 <version>${plugin.jetty.version}</version>
                 <configuration>
                     <webApp>
                         <contextPath>/</contextPath>
+                        <overrideDescriptor>src/test/webapp/WEB-INF/h2-console-web.xml</overrideDescriptor>
```

Comment:
The [file](https://github.com/DependencyTrack/dependency-track/pull/3730/files#diff-eca470f69b8d1376acb646175eee995ad2b84f99655a08c7867b896ec6f26c87) does not exist in the PR. And because we removed H2 entirely, I think you can simply omit this line.